### ISSUE TITLE
I really hope this is a bug with Mimic...

### ISSUE THREAD

**Issue by dirthsj** [1lann/Mimic#57]
![screen shot 2014-08-24 at 5 58 50 pm](https://cloud.githubusercontent.com/assets/3802881/4024944/5d2fc5aa-2be2-11e4-94bf-afe62566e11c.png)

I got above the rom directory O.O
Edit: Reproduction steps

```
> cd ....
..> cd ...
../.> dir
0
../.> cd 0
.././0> dir
```

**Comment by nevercast** [comment]
Probably related to https://github.com/1lann/Mimic/issues/51

**Comment by Fetchetch** [comment]
You can also edit the rom.... lol

**Comment by JerryWester** [comment]
Wow... If only this would work in the actual ComputerCraft... You'd be able to screw with other computers from the one you're using.

**Comment by Selim042** [comment]
That is probably the reason why it doesn't work in actual ComputerCraft.

**Comment by dirthsj** [comment]
Yes, I later confirmed this doesn't work in actual CC.  It doesn't seem like anyone actually pays attention to the issues here, and I'm pretty sure Mimic is dead at this point.
